src/vma/util/sys_vars.h

```diff
@@ -541,37 +541,37 @@ extern mce_sys_var & safe_mce_sys();
 #define MCE_DEFAULT_TCP_MAX_SYN_RATE                	(0)
 #define MCE_DEFAULT_TX_NUM_SEGS_TCP			(1000000)
 #define MCE_DEFAULT_TX_NUM_BUFS				(200000)
-#define MCE_DEFAULT_TX_NUM_WRE				(3000)
-#define MCE_DEFAULT_TX_NUM_WRE_TO_SIGNAL		(64)
+#define MCE_DEFAULT_TX_NUM_WRE				(256)
+#define MCE_DEFAULT_TX_NUM_WRE_TO_SIGNAL		(4)
 #define MCE_DEFAULT_TX_MAX_INLINE			(220) //224
 #define MCE_DEFAULT_TX_BUILD_IP_CHKSUM			(true)
 #define MCE_DEFAULT_TX_MC_LOOPBACK			(true)
 #define MCE_DEFAULT_TX_NONBLOCKED_EAGAINS		(false)
 #define MCE_DEFAULT_TX_PREFETCH_BYTES			(256)
-#define MCE_DEFAULT_TX_BUFS_BATCH_UDP			(8)
-#define MCE_DEFAULT_TX_BUFS_BATCH_TCP			(16)
+#define MCE_DEFAULT_TX_BUFS_BATCH_UDP			(1)
+#define MCE_DEFAULT_TX_BUFS_BATCH_TCP			(1)
 #define MCE_DEFAULT_TX_NUM_SGE				(2)
 #define MCE_DEFAULT_RX_NUM_BUFS				(200000)
-#define MCE_DEFAULT_RX_BUFS_BATCH			(64)
+#define MCE_DEFAULT_RX_BUFS_BATCH			(4)
 #define MCE_DEFAULT_RX_NUM_WRE				(16000)
-#define MCE_DEFAULT_RX_NUM_WRE_TO_POST_RECV		(64)
+#define MCE_DEFAULT_RX_NUM_WRE_TO_POST_RECV		(4)
 #define MCE_DEFAULT_RX_NUM_SGE				(1)
-#define MCE_DEFAULT_RX_NUM_POLLS			(100000)
+#define MCE_DEFAULT_RX_NUM_POLLS			(-1)
 #define MCE_DEFAULT_RX_NUM_POLLS_INIT			(0)
 #define MCE_DEFAULT_RX_UDP_POLL_OS_RATIO		(100)
 #define MCE_DEFAULT_RX_UDP_HW_TS_CONVERSION		(TS_CONVERSION_MODE_SYNC)
 #define MCE_DEFUALT_RX_SW_CSUM				(true)
 #define MCE_DEFAULT_RX_POLL_YIELD			(0)
 #define MCE_DEFAULT_RX_BYTE_MIN_LIMIT			(65536)
 #define MCE_DEFAULT_RX_PREFETCH_BYTES			(256)
-#define MCE_DEFAULT_RX_PREFETCH_BYTES_BEFORE_POLL	(0)
+#define MCE_DEFAULT_RX_PREFETCH_BYTES_BEFORE_POLL	(256)
 #define MCE_DEFAULT_RX_CQ_DRAIN_RATE			(MCE_RX_CQ_DRAIN_RATE_DISABLED)
-#define MCE_DEFAULT_GRO_STREAMS_MAX			(32)
+#define MCE_DEFAULT_GRO_STREAMS_MAX			(0)
 #define MCE_DEFAULT_TCP_3T_RULES			(false)
 #define MCE_DEFAULT_ETH_MC_L2_ONLY_RULES		(false)
-#define MCE_DEFAULT_SELECT_NUM_POLLS			(100000)
+#define MCE_DEFAULT_SELECT_NUM_POLLS			(-1)
 #define MCE_DEFAULT_SELECT_POLL_OS_FORCE		(0)
-#define MCE_DEFAULT_SELECT_POLL_OS_RATIO		(10)
+#define MCE_DEFAULT_SELECT_POLL_OS_RATIO		(0)
```

Comment:
This can cause a functionality issue for OOB.
Any application using select() which has 1 OS socket (management, logging, reporting...) over a non-offload NIC will not get any traffic! OS socket will be starved by VMA.